pkg/registry/apis/provisioning/jobs/pullrequest/worker.go

```diff
@@ -109,7 +109,7 @@ func (c *PullRequestWorker) Process(ctx context.Context,
 		return fmt.Errorf("read file: %w", err)
 	}
 
-	_, err = parser.Parse(ctx, fileInfo, true)
+	_, err = parser.Parse(ctx, fileInfo)
```

Comment:
I don't think we need to do a dry run to be able to generate some preview links.

Reply:
🤔  I would expec the results of this validation to be useful in the comment, but ya as implemented I think it would make a PR  with a red error box inside; and we are not (yet?) reporting the text errors in the github comment?